### PR TITLE
Main Menu work

### PR DESCRIPTION
**Pull request by dolanmiu**
# Validation success (button now blue)
image

# Validation Failed (Button Greyed out)
image

### CODE REVIEW

File: src/main/client/config/client-config.ts [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+   readonly name: string;
+}
+
 // Import this class for config properties, though, you should define here only development properties
 export class ClientConfig extends CommonClientConfig {
    static readonly ASSET_FOLDER: string = '../../src/asset/';
-   static readonly AUTO_START: boolean = true; // Skip main menu for development purposes
-   static readonly SERVER_HOST: string = `http://localhost:${ClientConfig.SERVER_PORT}`;
+   static readonly AUTO_START: boolean = false; // Skip main menu for development purposes
+   static readonly SERVER_HOSTS: ServerConfig[] = [
```
Comment (barnabasbartha) [inline src/main/client/config/client-config.ts]: There is already a ServerConfig on the server. We should use a name something like GameServer

Comment (dolanmiu) [reply src/main/client/config/client-config.ts]: ok will change

Comment (barnabasbartha) [reply src/main/client/config/client-config.ts]: Or ServerHost as the variable name suggests

Comment (barnabasbartha) [reply src/main/client/config/client-config.ts]: Or rather ServerInstance

**Review by barnabasbartha — APPROVED** [review]

**Pull request merged**